lib/config/error-codes/api.json

```diff
@@ -160,9 +160,14 @@
           "class": "UnauthorizedError"
         },
         "forbidden_action": {
-          "code": 6,
+          "code": 7,
```

Comment:
I thought we had a check at startup ensuring that there were no code duplicates in:
* domain codes
* subdomain codes within a same domain
* error codes within a same subdomain

This change shows that we definitely MUST have one: we cannot change error codes on a whim, once they are released, they must be pretty much unchangeable.

Reply:
The check at startup is about the format of the json file
I ll update it